changelog/fix-bump-min-php-wc-beta

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+Update automated test matrix to remove PHP 7.3 tests against WooCommerce beta.
```

Comment:
For dev patches, you can skip the message and just add a comment, similar to this: https://github.com/Automattic/woocommerce-payments/blob/1b6d3519a8d1f6c0906b32e849bbfcd49d32652d/changelog/rpp-basic-process. By doing so, the entry will not be included in the release changelog, which makes sense in this case, as the change will not be released to merchants.